streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -947,7 +947,7 @@ private void addStreamTasks(final Collection<TopicPartition> assignment, final l
                 try {
                     final StreamTask task = findMatchingSuspendedTask(taskId, partitions);
                     if (task != null) {
-                        log.debug("{} recycling old task {}", logPrefix, taskId);
+                        log.debug("{} resuming old task {}", logPrefix, taskId);
```

Comment:
nit: this is not introduced in this PR but, other places capitalize the first letter after log prefix.